packages/suite/src/components/wallet/TokenSummaryIconWrapped.tsx

```diff
@@ -0,0 +1,32 @@
+import { selectCoinDefinitions } from '@suite-common/token-definitions';
+import {
+    enhanceTokensWithRates,
+    getTokens,
+    sortTokensWithRates,
+} from 'src/utils/wallet/tokenUtils';
+import { TokenSummaryIcon } from '@trezor/product-components';
+import { useSelector } from 'src/hooks/suite';
+import { selectLocalCurrency } from 'src/reducers/wallet/settingsReducer';
+import { selectCurrentFiatRates } from '@suite-common/wallet-core';
+import { Account } from '@suite-common/wallet-types';
+
+type TokenSummaryIconWrappedProps = {
+    account: Account;
+};
+
+export const TokenSummaryIconWrapped = ({ account }: TokenSummaryIconWrappedProps) => {
```

Comment:
```suggestion
export const TokenSummaryIconWrapper = ({ account }: TokenSummaryIconWrappedProps) => {
```

but btw I am thinking if `TokenSummaryIcon` is good naming. Probably not? 😀

Reply:
`TokenIconBundle`, `TokenIconCluster`, `TokenIconStack`,...

Reply:
I agree and I like TokenIconSet the most

Reply:
Wrapper please

Reply:
Sorry missed this one, done